### PR TITLE
Encrypt secrets with kms

### PR DESCRIPTION
**Pull request by jacek-adamek**

### CODE REVIEW

File: lib/devise_google_authenticator.rb [hunk 1]
```diff
@@ -26,6 +26,9 @@ module Devise # :nodoc:
 
   mattr_accessor :ga_skip_validation_if
   @@ga_skip_validation_if = false
+
+  mattr_accessor :ga_kms_key_name
+  @@ga_kms_key_name = nil
```
Comment (marvin5064) [inline lib/devise_google_authenticator.rb:31]: How this been specified from app who use this package

Comment (jacek-adamek) [reply lib/devise_google_authenticator.rb:31]: Here https://github.com/bitgaming/gaming_admin/blob/b8acfb89cdf332adbf9fb582c14a79ed1322044b/config/initializers/devise.rb#L15 will add sth like 
```
  config.ga_kms_key_name = 'haha'
```
Actually I have something like above locally

Comment (marvin5064) [reply lib/devise_google_authenticator.rb:31]: Cool, as long as client could skip it for backwards compatibility. It should be fine

**Review by marvin5064 — APPROVED** [review]
Thanks

**Comment by marvin5064** [comment]
One part, if it s version 1, and missing kms setup we should error early

**Review by marvin5064 — APPROVED** [review]
Thanks

**Pull request merged**